setup.cfg

```diff
@@ -16,6 +16,8 @@ multi_line_output = 3
 filterwarnings = error
     ignore:numpy.ufunc size changed,:RuntimeWarning
     ignore:Using a non-tuple sequence:FutureWarning
+    ignore:numpy.linalg support is experimental:UserWarning
+    ignore:scipy.linalg support is experimental:UserWarning
```

Comment:
mvn triangular_solve / cholesky / eigh require numpy.linalg, so we need to filter those warnings.

Reply:
Could you remove `@pytest.mark.filterwarnings` from `test_mcmc` since we don't need that now?